.github/workflows/purview-recipe-ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Microsoft Purview Recipe CI
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - src/az-purview/deploy/**
+      - .github/workflows/purview-recipe-ci.yml
+  pull_request:
+    branches: [main]
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  AZURE_REGION: eastus
+
+jobs:
+  validate-bicep:
+    runs-on: ubuntu-latest
+    name: Validate Terraform
+    environment: ${{ github.event.repository.name}}
```

Comment:
I still this property on other bicep deployment sample.

Reply:
It would seem that I failed to remove the "environment" property on the _validate-bicep_ task. The _validate-terrafrom_ task should not have "environment". The "environment" property was previously used for some shenangins related to using Azure credentials. Since the workflows are no longer using Azure credentials, the use "environment" isn't necessary. 

Looks like I've got a PR in my future!  Thanks for flagging this one!  :)